src/components/matomo/Matomo.js

```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types'
+import queryString from 'query-string'
+
+const Matomo = ({ location }) => {
+  const Matomo = window._paq
+  const searchParameters = queryString.parse(location.search)
+
```

Comment:
J'aurai introduit les variables search et pathname via destructuration
```suggestion
const { pathname, search } = location
```
WDYT ?